src/sidebar/cross-origin-rpc.js

```diff
@@ -48,7 +51,18 @@ function isJsonRpcMessage(data) {
 function start(store, settings, $window) {
   const methods = registeredMethods(store);
 
-  $window.addEventListener('message', function receiveMessage(event) {
+  // Process the pre-start incoming RPC requests
+  preStartQueue.forEach(event => {
+    receiveMessage(event);
+  });
+  // Clear the queue and remove the preStartMessageListener
+  preStartQueue = [];
+  window.removeEventListener('message', preStartMessageListener);
+
+  // Start listening to new RCP requests
```

Comment:
```suggestion
  // Start listening to new RPC requests
```

I noticed this easily-made typo in a few other places. It might be helpful to install an add-on for your editor to catch this?